commands/motor/MotorSet.java

```diff
@@ -44,24 +44,26 @@ public void set(double speed) {
 	}
 
 	@Override
-	protected void execute() {
+	public void execute() {
 		motor.set(speed);
 		LogKitten.d("MotorSet executing with speed " + speed);
 	}
 
 	@Override
-	protected void end() {
+	public void end(boolean interrupted) {
 		motor.set(0);
 		LogKitten.d("MotorSet ended (motor speed set to 0)");
 	}
 
 	@Override
-	protected void interrupted() {
-		LogKitten.d("MotorSet interupted (motor speed undefined)");
```

Comment:
before, it would log when the command was interrupted, and you just removed that. it should probably be added back inside the end() method as `if(interrupted) {LogKitten.d("Motorset INterrupted ... ")}`